catalog/core/catalog-core-commands/src/test/java/org/codice/ddf/commands/catalog/SeedCommandTest.java

```diff
@@ -205,6 +212,19 @@ public void testDoesNotDownloadCachedResource() throws Exception {
         verify(catalogFramework, times(2)).query(any(QueryRequest.class));
     }
 
+    @Test
+    public void testDoesNotSkipResources() throws Exception {
+
+        int maxPageSize = NumberUtils.toInt(System.getProperty("catalog.maxPageSize"), 1000);
+        int resourceLimit = maxPageSize * 2;
```

Comment:
❓ Why is this limit set to double the maxPageSize? Could this be any value, or is 2 special? Something to include in the JavaDoc to explain the methodology of this test.